packages/core/client.js

```diff
@@ -58,6 +58,10 @@ class Client {
     this.Breadcrumb = Breadcrumb
     this.Session = Session
 
+    // store the time we were constructed so we can calculate the duration that
+    // the application has been running when an error is thrown
+    this._startTime = new Date()
```

Comment:
Would this be better outside the class, at the module level (as it was with expo) in order to give a more accurate start time? We suggest people start bugsnag first so in the simple case it likely won't make much difference. But for more complex cases the difference between the javascript being parsed and a client being newed could be significant.